protected/humhub/modules/like/permissions/CanLike.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2020 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\like\permissions;
+
+use humhub\modules\space\models\Space;
+
+/**
+ * CanLike Permission
+ */
+class CanLike extends \humhub\libs\BasePermission
+{
+
+    /**
+     * @inheritdoc
+     */
+    public $defaultAllowedGroups = [
+        Space::USERGROUP_OWNER,
+        Space::USERGROUP_ADMIN,
+        Space::USERGROUP_MODERATOR,
+        Space::USERGROUP_MEMBER,
+    ];
+
+    /**
+     * @inheritdoc
+     */
+    protected $fixedGroups = [
+        Space::USERGROUP_GUEST,
```

Comment:
I think also non space members should be able to like.

The standard behavior should not change after the PR. It should only be an additional option to deny likes.

Reply:
Please take a look at the[ comment above](https://github.com/humhub/humhub/pull/4339#discussion_r482096038).
I need some help with this. what could be  wrong  with it (Why I am not able to like my posts and in some other cases)

Reply:
@MasterMindNET  This should solves the problem with the user profile related likes:

```
+++ b/protected/humhub/modules/like/permissions/CanLike.php
@@ -9,6 +9,7 @@
 namespace humhub\modules\like\permissions;

 use humhub\modules\space\models\Space;
+use humhub\modules\user\models\User;

 /**
  * CanLike Permission
@@ -20,6 +21,9 @@ class CanLike extends \humhub\libs\BasePermission
      * @inheritdoc
      */
     public $defaultAllowedGroups = [
+        User::USERGROUP_SELF,
+        User::USERGROUP_FRIEND,
+        User::USERGROUP_USER,
         Space::USERGROUP_OWNER,
         Space::USERGROUP_ADMIN,
         Space::USERGROUP_MODERATOR,
```

Reply:
Oh, Thank you so much for this, I could not find it before 😃 
Please advise me which lines I should remove and which I should leave as a default?

I am really happy that everything is working just fine, but these lines were a real solution.

Reply:
Oh, Thank you so much for this, I could not find it before 😃 
Please advise me which lines I should remove and which I should leave as a default?

I am really happy that everything is working just fine, but these lines were a real solution.

I am looking forward to hearing from you follow-up instructions